WatchApp Extension/Managers/ComplicationChartManager.swift

```diff
@@ -117,15 +118,50 @@ final class ComplicationChartManager {
     }
 
     private func drawOverridesIfNeeded(in context: CGContext, using scaler: GlucoseChartScaler) {
-        guard
-            let override = data?.activeScheduleOverride,
-            let overrideHashable = TemporaryScheduleOverrideHashable(override)
-        else {
-            return
+        let overrideColor = UIColor.glucose.withAlphaComponent(0.4).cgColor
+        let extendedOverrideColor = UIColor.glucose.withAlphaComponent(0.25).cgColor
+        let spannedInterval = scaler.dates
+
+        func drawOverride(
+            _ override: TemporaryScheduleOverride,
+            pushingStartTo startDate: Date? = nil,
+            extendingToChartEnd shouldExtendToChartEnd: Bool
+        ) {
+            var override = override
```

Comment:
[nit] `override` use confuses syntax highlighter thinking it is the `override` keyword.

Reply:
GitHub's syntax highlighting fails to support Swift's contextual keywords. Xcode does not have this problem; leaving as-is.